WinUIGallery/Controls/ControlExample.xaml

```diff
@@ -138,26 +160,36 @@
                 <muxc:Expander.Header>
                     <TextBlock Text="Source code" />
                 </muxc:Expander.Header>
+                <Grid>
```

Comment:
```suggestion
                <Grid RowSpacing="8">
```
If it were me, I would add just a few pixels between the SelectorBar and the Grid.